stores/sql/sqlite/main.go

```diff
@@ -1071,14 +1071,23 @@ func (tx *MainDatabaseTx) UpdateHostBlocklistEntries(ctx context.Context, add, r
 			return fmt.Errorf("failed to prepare insert statement: %w", err)
 		}
 		defer insertStmt.Close()
+
 		joinStmt, err := tx.Prepare(ctx, `
 		INSERT OR IGNORE INTO host_blocklist_entry_hosts (db_blocklist_entry_id, db_host_id)
-		SELECT ?, id FROM (
+		SELECT $1, id FROM (
```

Comment:
`nit` use `?` the dollar placeholder syntax is used in PostgreSQL but for SQLite and MySQL the `?` is more idiomatic